docs/release/release_template.rst

```diff
@@ -12,31 +12,29 @@ For more information, examples, and documentation, please visit our website:
 
 https://github.com/napari/napari
 
-
 New Features
-------------
-
-
+************
 
 Improvements
-------------
-
+************
 
+Bugfixs
+*******
 
 API Changes
------------
-
-
-
-Bugfixes
---------
-
-
+***********
 
 Deprecations
-------------
+*************
 
+Build Tools
+***********
 
+Other Pull Requests
+*************
```

Comment:
Try to line up the underline length to the text. I've seen sphinx complain about this in the past.

Reply:
fixed